pkg/controller/traceflow/controller.go

```diff
@@ -67,8 +71,9 @@ type Controller struct {
 }
 
 // NewTraceflowController creates a new traceflow controller.
```

Comment:
In this comment, maybe worth mentioning that a new "podIP" indexer is added to the podInformer, since it may affect other pieces of the code also using this informer?

Reply:
Change to:
// NewTraceflowController creates a new traceflow controller and adds podIP indexer to podInformer.